Antonio_Ramos_Documentation

```diff
@@ -140,12 +148,86 @@ new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Ba
+- after communicating with Will, we were still not able to get documentation which aligns with the data referenced in the python scripts. Documentation that Will sent over was the same as what was already in the github repository. File not downloaded since it is the same as what is in the git repo. 
+
+- finished filling in Data & Script File Table which contains a list of all the files mentioned by Will Heikoff in his documentation, or utilized by the python script "K-function Local_3Stage.py" saved it in G:\MAX-Filler\Collab\Labs-kbuzard-S18\Admin . 
+
+- will now go through the following programs which Will Heikoff wrote to change aspects of the data so that it would work with the python scripts.
+1. ZIPS_Tot_Emp.r 
+2. CA_WAC_Transformation.R
+3. ZCTA_Manufa_Trans.R
+4. ZCTA_GoodLabs_Trans.py
+5. ZCTA_Manufa_Trans.py 
+6. GoodLabs_Trans.py
+(more complete descriptions avaliable in Data & Script File Table located at G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin) 
+
+- discovered that the two of the three data files, GoodLabs_New.shp and CA_ZCTA_Man.shp, from the K-function Local_3Stage.py script come from one python script, "ZCTA_GoodLabs_Trans.py" (located at "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\ZCTA_GoodLabs_Trans.py") 
+
+- ZCTA_GoodLabs_Trans.py changes the CRS for GoodLabs.shp and exports GoodLabs_New.shp 
+
+- ZCTA_GoodLabs_Trans.py also also reads a csv file (DEC_00_SF3_DP3_with_ann.csv) and joins it with an imported shapefile (tl_2010_06_zcta500.shp) changes the CRS and then exports CA_ZCTA_Man.shp
+
+- all other R and python scripts do not seem to be relevant to the data that is currently being used by the K-function Local_3Stage.py script. However, it is not clear where the data file CA_Block_New.shp came from which is the third data file being read into the K-function Local_3Stage.py script. 
+
+- In the folder located at "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Block Level Analysis" I found two programs that might be related to CA_Block_New.shp 
+ 1. CA_Block_Trans.py
+ 2. CA_WAC_Trans.py 
+both programs are written August 8th but there is no documentation for why 
+
+- CA_Block_Trans.py has a note that says "Need to figure out how to change shapefile from ANSI 1252 to UTF-8 encoding." Not sure if this was ever done will look for more up to date file. After examining the code it looks like CA_Block_Trans.py is the same as CA_WAC_Trans.py except CA_Block_Trans.py changes the geometry and CRS. CA_Block_Trans.py reads in a csv.gz file (ca_wac_S000_JT00_2002.csv.gz) and and joins it with an imported shapefile (CA_block_2000.shp) then changes the geometry and CRS and exports a shapefile (CA_2000_Tot_Emp.shp) however, this is not the block data file which is being read into K-function Local_3Stage.py. 
+
+- found the origin of ca_wac_S000_JT00_2002.csv.gz, https://lehd.ces.census.gov/data/lodes/LODES7/ca/wac/  (ask prof. buzard if she can help clarify what the content of this file is) 
```

Comment:
I will look into this first thing tomorrow morning.

Reply:
I found the documentation pretty easily by shortening the URL a little at a time (that's a trick that often works). It's in the folder https://lehd.ces.census.gov/data/lodes/LODES7/

If you click on the "ca" folder for California, there's a version.txt file that says that the California data is version 7.4. Looking at the dates for the file we need, it might be 7.3, but I don't think there are any changes that affect the variable we need.

So the document at https://lehd.ces.census.gov/data/lodes/LODES7/LODESTechDoc7.4.pdf will work.

As you'll see, the LODES (LEHD Origin-Destination Employment Statistics) data is pretty amazing. We're using the WAC (Workplace Area Characteristics) because we want to know the location where people actually work. Then it's a matter of finding the earliest file we can (2002--we'd rather have 1998, but this data program didn't start until 2002) for all workers (**S000**) and all jobs (**JT00**). The only variables we care about in this big dataset (described on page 9) are the block identifier (**w_geocode**) and the total number of jobs (**C000**). We use manufacturing jobs at the zip code level, but total jobs at the block level.